app/assets/stylesheets/govuk-component/_breadcrumbs.scss

```diff
@@ -34,3 +35,11 @@
     }
   }
 }
+
+.govuk-breadcrumbs--inverse a {
+  color: $white;
+}
+
+.breadcrumb-for-current-page--inverse a {
```

Comment:
This is being applied to the `a` element, so we don't need the `a` part of the selector.